account_payment_mode/models/account_payment_method.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# © 2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields, api
+
+
+class AccountPaymentMethod(models.Model):
+    _inherit = 'account.payment.method'
+    _rec_name = 'display_name'
+
+    code = fields.Char(
+        string='Code (Do Not Modify)',
+        help="This code is used in the code of the Odoo module that handle "
+        "this payment method. Therefore, if you change it, "
+        "the generation of the payment file may fail.")
+    active = fields.Boolean(string='Active', default=True)
+    # In the pain_base module, we will add a default_pain_version
+    display_name = fields.Char(
```

Comment:
The 'name' field of account.payment.method is not translatable, see https://github.com/odoo/odoo/blob/9.0/addons/account/models/account_payment.py#L24